### DISCUSSION TITLE
Suggestion/REQUEST - 'Back gifs' of main pokemon should be changed (or added as an option to choose b/w) to 'right side/sideways gif' as it will be a lot more appealing (imo)

### DISCUSSION THREAD

**Discussion by CaptainDead** [Unlucky-Life/ankimon#72] [Ideas]
I think that it would be better if the 'back images' of Pokemon were not showing their backs and looking inside the screen.
Because it doesn't look that good, we spend most of the time with our Main pokemon and if it doesn't look good its somewhat affects the fun of it all. Like look at them - 
![19](https://github.com/Unlucky-Life/ankimon/assets/139457914/fef55e81-be51-49b9-a5c6-ff7b1a94c589)
![20](https://github.com/Unlucky-Life/ankimon/assets/139457914/7bbbd09f-7058-482a-a312-e7ac73c2a12f)
![40](https://github.com/Unlucky-Life/ankimon/assets/139457914/7109a813-25e6-4cb0-be66-6f418acbf33b)
![45](https://github.com/Unlucky-Life/ankimon/assets/139457914/79c6426e-1aac-43d0-89f7-527c8a00aa3d)
![55](https://github.com/Unlucky-Life/ankimon/assets/139457914/a6c4e050-8cbe-4478-a5ec-5abfd0d6111f)

Majority of them are not that fun to look at when they are looking inside the screen.
But the same one looks great when we can see their front - 
![19](https://github.com/Unlucky-Life/ankimon/assets/139457914/4574724f-8d4b-4b76-8169-c034e7e1a09c)
![20](https://github.com/Unlucky-Life/ankimon/assets/139457914/9d1919f4-345e-4963-88c9-92a9674961d4)
![40](https://github.com/Unlucky-Life/ankimon/assets/139457914/76675616-f2b2-4d3e-a7f8-6c61b990f722)
![45](https://github.com/Unlucky-Life/ankimon/assets/139457914/22cd0964-ff37-4bc9-b54c-058696daddb9)
![55](https://github.com/Unlucky-Life/ankimon/assets/139457914/32632d01-5f3d-4d33-ae02-a92867972f85)

Them looking inside the screen looks great in the ankimon windows, perfect. 
![image](https://github.com/Unlucky-Life/ankimon/assets/139457914/750003ee-bdea-43de-9c74-5fa95fafc3b0)

But for reviewer, I believe that it would be better if our main pokemon were looking right side, our opponents (front looking gifs above) all look towards the left side so it would look like a battle as well if our main pokemon was looking towards right side.

Something like this -
![tumblr_pe28p0GLff1r8r6mfo1_500](https://github.com/Unlucky-Life/ankimon/assets/139457914/29824a20-ce89-4ab0-b267-32458f290ac0)

One of the Back Gifs somewhat looks like this - 
![4](https://github.com/Unlucky-Life/ankimon/assets/139457914/b7ee8337-ef2d-43c4-add6-7acfcdccd56f)

This one is not completely looking into the screen and more sideways and thus looks a lot better, and this is my main pokemon for this reason as other pokemon I have doesn't look that cool when they look into the screen - 

This pokemon when he was my opponent - "Wow he looks so good, can't wait to catch it!"
![561](https://github.com/Unlucky-Life/ankimon/assets/139457914/418e100f-2bf6-4ca7-bec3-63a74748339e)

Pokemon after I've caught it - "ahh not looking that good anymore, its fine ig"
![561](https://github.com/Unlucky-Life/ankimon/assets/139457914/d5b1e4b7-72a2-4942-b0d6-c4103e65123f)

Not tryna say the ones that are here are that bad, they're great, I love them and this addon but I think it would look better if they were looking right side.

However, I don't know how u got all these gifs, but if there is a source from where you can get right-side-looking gifs of all Pokemons without crazy efforts (like looking for gifs of all of them on the internet one by one) then I think you should give them a try too.

And if some people prefer their main pokemons to be looking inside the screen then you can give us an option in config to choose b/w back gifs and sideways gifs for our main pokemons.

I think if the GIFs were looking sideways (right side) and if we can see their front side and face, it will be a LOT more fun and cool, and would increase the enjoyment of the addon.

Thanks.

**Comment by Unlucky-Life** [comment]
@CaptainDead I think this is a good idea, and i think this can be implemented easily

**Reply by Unlucky-Life** [reply]
![image](https://github.com/Unlucky-Life/ankimon/assets/77027147/6fe0cf1d-5ef6-400d-98aa-d36a0667e744)
Something like this right ?

**Reply by CaptainDead** [reply]
yeah ig this is perfect
